internal/providers/terraform/aws/instance.go

```diff
@@ -35,27 +42,75 @@ func computeCostComponent(d *schema.ResourceData, region string, purchaseOption
 		"spot":      "spot",
 	}[purchaseOption]
 
-	return &schema.CostComponent{
-		Name:           fmt.Sprintf("Compute (%s, %s)", purchaseOptionLabel, instanceType),
-		Unit:           "hours",
-		HourlyQuantity: decimalPtr(decimal.NewFromInt(1)),
-		ProductFilter: &schema.ProductFilter{
-			VendorName:    strPtr("aws"),
-			Region:        strPtr(region),
-			Service:       strPtr("AmazonEC2"),
-			ProductFamily: strPtr("Compute Instance"),
-			AttributeFilters: []*schema.AttributeFilter{
-				{Key: "instanceType", Value: strPtr(instanceType)},
-				{Key: "tenancy", Value: strPtr(tenancy)},
-				{Key: "operatingSystem", Value: strPtr("Linux")},
-				{Key: "preInstalledSw", Value: strPtr("NA")},
-				{Key: "capacitystatus", Value: strPtr("Used")},
+	costComponents := []*schema.CostComponent{
+		{
+			Name:           fmt.Sprintf("Linux/UNIX Usage (%s, %s)", purchaseOptionLabel, instanceType),
```

Comment:
Linux/UNIX Usage -> Linux/UNIX usage ?

See `upper/lower case` in https://github.com/infracost/infracost/blob/master/CONTRIBUTING.md, we can change that if we like though.

Reply:
👍 will fix this and other resources to be consistent